build_tools/ci/run_matmul_test.sh

```diff
@@ -206,19 +206,22 @@ function run_matmul_test() {
   # The default is to not expect a compilation failure.
   local expect_compile_failure="0"
 
-  # The default is to compile and run the numerical test.
-  local compile_only="0"
-
   local do_transpose_rhs="0"
 
   # The maximum number of elements to check for correctness.
   # See https://github.com/iree-org/iree/blob/tools/testing/e2e/test_utils.c#L40-L47
   local max_elements_to_check="20000"
 
+  # After compilation, the test with be run 'n_repeats' times. This option (when
+  # set greater than 1) is useful for shapes which might be 'flakey' and fail
+  # intermittently. It is also useful if a test is know to fail at runtime but
+  # should still be checked to compile (set n_repeats=0 in this case).
```

Comment:
Most of the time the intermittent failure is caused by compilation problem not runtime. It's not helpful to just compile once and run multiple times.